Assets/Scripts/Weapons/BurstShooter.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System.Collections;
+using System.Collections;
```

Comment:
Wtf is your IDE even doing with all of these random edits? lol

Reply:
Dude. I have no idea...